### PR TITLE
IS-686: Created getting started docker file with agents already set up

### PR DESCRIPTION
**Pull request by stevengubler**
I created a dockerfile that would automatically setup the environment for the getting started guide.

- `README.md` explains how to use the dockerfile
- `alice_agent_pool.dockerfile` is the dockerfile. (based off of ci/indy-pool.dockerfile)
- `agents_setup.sh` is the script the container runs to start and configure the pool
- `getting_started.indyscript` is the indy-cli batch file that configures the pool
- `node_supervisord.conf` was built into ci/indy-pool.dockerfile, but I refactored it out because I think it is easier to deal with this way.

Hopefully, this can be used to significantly simplify the getting started guide or something like that. I leave that up to @michaeldboyd or whoever wants to do something like that.

### CODE REVIEW

**Comment by vimmerru** [comment]
For the current moment GSG is available as:

1. Libindy sample: indy-sdk\samples\python\src\getting_started.py
2. Jupyter script: C:\workspace\src\indy-sdk\doc\getting-started\getting-started.ipynb

- To run 1 you just need the pool from indy-sdk\ci\indy-pool.dockerfile
- To run 2 we already have docker-compose script that starts all environment

I don't understand why we need something additional.

File: doc/getting-started/agents_utils/agents_setup.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+LOGS_DIR=/home/indy/logs
+
+mkdir -p $LOGS_DIR
+
+startagent () {
+    local NAME=$1
+    local PORT=$2
+    echo Starting agent \"$NAME\" on port $PORT
+    python3 "/usr/local/lib/python3.5/dist-packages/indy_client/test/agent/$NAME.py" --port $PORT > $LOG_DIR/$NAME.log
```
Comment (jovfer) [inline doc/getting-started/agents_utils/agents_setup.sh]: @stevengubler @ashcherbakov is it script for running old, python-based Agent for getting started?

**Comment by jovfer** [comment]
@stevengubler If you reference to something (https://github.com/hyperledger/indy-sdk/pull/747#issuecomment-392911047), please provide the link, not just quoted text without context of task, conversation, etc.

**Comment by vimmerru** [comment]
@jovfer @ashcherbakov @stevengubler 

I see that we are trying to add some code for old deprecated GSG that is still supported in indy-node repo. For me it looks like it will confuse people a lot. Also we already have similar setup here https://github.com/hyperledger/indy-node/blob/master/environment/docker/pool/StartIndyAgents.md

**Comment by stevengubler** [comment]
@jovfer @ashcherbakov @vimmerru 
Oh good. I was unware of these existing docker setups. Maybe work can be done to make them a little more visible and intuitive. Either way, I guess it makes this PR unneeded. Thanks for your help.

---
*Also, as a matter of etiquette, should I close this pull request or let you guys do it?*

**Comment by jovfer** [comment]
@stevengubler 
> should I close this pull request or let you guys do it?
Usually we wait consensus from both sides: the author and reviewer. As I see you opinion to close the PR I will do it. Personally I have only one suggesting: if you are closing your PR by yourself please provide the reason in comment.

> And one last question, do you guys happen to know off the top of your head where client.ubuntu.dockerfile gets the indycore base image? Is that on docker hub?
AFAIR `indycore` built from neighbor dockerfile. BTW it's a good question for https://chat.hyperledger.org/channel/indy-node because answer may be helpful for community

**Pull request closed without merging**